packages/common/src/core/getEntityDefaultWorkspacePane.ts

```diff
@@ -0,0 +1,6 @@
+export const getEntityDefaultWorkspacePane = (entityType: string): string => {
```

Comment:
We'll need to consume this method from `opendata-ui`, so we'll need to make sure it is exported by `@esri/hub-common`. To do that, I believe we need to re-export it from [index.js](https://github.com/Esri/hub.js/blob/8529/8527/workspace-pane-routing/packages/common/src/core/index.ts) in the `core` directory